Paws4Me/Paws4Me/Data/LocalPetNameViewController.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  LocalPetNameViewController.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/03/10.
+//
+
+import CoreData
+import UIKit
+
+class LocalPetViewController: UIViewController {
+    // MARK: - IBOutlets
+    @IBOutlet weak var petNameTableView: UITableView!
+
+    // MARK: - Vars/Lets
+    var namePet = ""
+    var imagePet = ""
+    var pets: [NSManagedObject] = []
+
+    // MARK: - Life cycle
+    override func viewDidLoad() {
+      super.viewDidLoad()
+      title = "Saved Pets"
+        petNameTableView.register(UITableViewCell.self, forCellReuseIdentifier: "Cell")
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+      super.viewWillAppear(animated)
+
+      guard let appDelegate = UIApplication.shared.delegate as? AppDelegate else {
+        return
+      }
+
+      let managedContext = appDelegate.persistentContainer.viewContext
```

Comment:
Rather put this code in a separate functions, code like this doesn't really belong in the `viewWillAppear`